src/test/java/uk/gov/pay/directdebit/payments/dao/PaymentDaoIT.java

```diff
@@ -20,14 +20,19 @@
 import java.sql.Date;
 import java.sql.Timestamp;
 import java.time.LocalDate;
+import java.time.Month;
 import java.time.ZonedDateTime;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 import java.util.Set;
 
+import static java.time.Month.*;
```

Comment:
I'm not where I stand on wildcard imports but GDS doesn't like them: https://gds-way.cloudapps.digital/manuals/programming-languages/java.html#imports

Reply:
sorry.... I've increased my intelij settings up from 99.....with all the 9's.